admin-dev/themes/default/js/bundle/product/product-combinations.js

```diff
@@ -277,3 +278,15 @@ var combinations = (function() {
 BOEvent.on("Product Combinations Management started", function initCombinationsManagement() {
   combinations.init();
 }, "Back office");
+
+/**
+ * Refresh bulk actions combination number after creating or deleting combinations
+ *
+ * @param {number} sign
+ * @param {number} number
+ */
+var refreshTotalCombinations = function (sign, number) {
```

Comment:
After further analysis, this method should be owned by `admin-dev/themes/new-theme/js/product-page/product-bulk-combinations.js`, but if we moved it there then it wouldn't be accessible from here.

Ideally, an event should be emitted when adding/removing combinations, and bulkCombinations would listen to it in order to update the number of combinations whenever a combination is added/removed.

However, the current state of things doesn't allow it without refactoring, so unfortunately this will have to do.